ZenPacks/zenoss/Import4/bin/import4

```diff
@@ -24,19 +25,41 @@ exitCode = {
 def run_migration(migration_class, args):
     # The main control running a migration process
     _migration = migration_class(args, progress_report_callback)
-    result = _migration.prevalidate()
-    if args.dryrun or isFailure(result, args.ignorewarnings):
-        exit(exitCode[result])
+
+    # the prevalidation phase
+    try:
+        _migration.prevalidate()
+    except migration.ImportError as err:
+        sys.stderr.write('Error: %s(%d)\n' % (err.error_string, err.return_code))
+        _rc = exitCode[err.error_string]
+        if _rc >= 2:
```

Comment:
Yup.. now the fixed numbers are changed to more descriptive exitCode definitions.